plugins/module_utils/network/common/rm_base/network_template.py

```diff
@@ -8,12 +8,20 @@
 from ansible_collections.ansible.netcommon.plugins.module_utils.network.common.utils import (
     Template,
     dict_merge,
+    validate_config as _validate_config,
 )
-
 from ansible_collections.ansible.netcommon.plugins.module_utils.network.common.rm_base.resource_module_base import (
     RmEngineBase,
 )
 
+try:
+    # this method was renamed in 2.11 devel
```

Comment:
Is this the new location or the old location? Whenever we get to go back and clean this up, I would like to not have to figure out which to remove. I think something like `TODO: Remove this import when we no longer support ansible < 2.11` would be easier to parse and also to find later on

Reply:
+1. Will update the comment.